chaosinstana/api.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+
+from chaoslib.types import Configuration, Secrets
+from logzero import logger
+import requests
+
+
+__all__ = ['call_api', 'get_all_events']
+
+
+def call_api(path: str, configuration: Configuration,
+             secrets: Secrets, query: Dict[str, Any] = None,
+             body: Dict[str, Any] = None, method: str = "GET",
+             headers: Dict[str, str] = None) -> requests.Response:
+    """
+    TBD
+    """
+    # location given here
+    URL = "https://httpbin.org/get"
+
+    # defining a params dict for the parameters to be sent to the API
```

Comment:
Personally I think these documentation messages can be removed as the code is clear enough?